### PR TITLE
fix(curriculum): update directions and hint text for step 117 of RPG game

### LINKED ISSUE #53502: Update the directions and hint text for step 117 for RPG game
### Describe the Issue

We have a lot of campers struggling with how to do step 117 of the RPG game on the forum.
https://forum.freecodecamp.org/search?q=step%20116%20%23javascript

There is still some confusion on how the monsters array works and how to access information from it.

It would be best to update the directions and hint text to more explicitly show them what needs to be done. 

## Updated step 117 directions

```md
# --description--

Now, you will need to update the text for the current monster's name and health.

Start by assigning `monsters[fighting].name` to the `innerText` property of `monsterName`. Then, assign `monsterHealth` to the `innerText` property of `monsterHealthText`.
```
### updated step 117 hints

I didn't make any changes to the tests because those are fine.

Here are the updates to the hint text only for hints 2 and 4

````md
# --hints--

You should use dot notation to access the `innerText` property of `monsterName`.

```js
assert.match(goFight.toString(), /monsterName\.innerText/);
```

You should assign `monsters[fighting].name` to `monsterName.innerText`.

```js
assert.match(goFight.toString(), /monsterName\.innerText\s*=\s*monsters\s*\[\s*fighting\s*\]\s*\.name/);
```

You should use dot notation to access the `innerText` property of `monsterHealthText`.

```js
assert.match(goFight.toString(), /monsterHealthText\.innerText/);
```

You should assign `monsterHealth` to `monsterHealthText.innerText`.

```js
assert.match(goFight.toString(), /monsterHealthText\.innerText\s*=\s*monsterHealth|monsterHealthText\.innerText\s*=\s*monsters\s*\[\s*fighting\s*\]\s*\.health/);
```
````

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-117

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by itsmacr8**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #53502

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
These are not the changes required by the linked issue. Please take another look at the post https://github.com/freeCodeCamp/freeCodeCamp/issues/53502#issue-2112928775

**Comment by itsmacr8** [comment]
I have updated the code.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e271f8e3d1541f9624ad.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-117
 
 # --description--
 
-Now, set the `innerText` property of `monsterName` to be the `name` property of the current monster. Do the same for `monsterHealthText` and the `health` property.
+Now, Start by assigning `monsters[fighting].name` to the `innerText` property of `monsterName`. Then, assign `monsterHealth` to the `innerText` property of `monsterHealthText`.
```
[the remark this answers is not in the document]
Comment (itsmacr8) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e271f8e3d1541f9624ad.md:10]: Okay.

**Review by gikf — APPROVED** [review]

**Review by JoyShaheb — APPROVED** [review]
LGTM 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by itsmacr8** [comment]
Thank you so much for your support and help.

**Pull request merged**